apps/fluent-tester/src/FluentTester/testPages.web.ts

```diff
@@ -61,4 +62,9 @@ export const tests: TestDescription[] = [
     component: PersonaTest,
     testPage: HOMEPAGE_PERSONA_BUTTON,
   },
+  {
+    name: 'Tokens Test',
+    component: TokenTest,
+    testPage: HOMEPAGE_TOKEN_BUTTON,
+  },
```

Comment:
Could we add this to all the platforms? Or will it not work / hasn't been tested yet? I can test mobile + macOS later if that helps

Reply:
I think it'll work? I just didn't verify outside of win32. I can add it to other platforms and test uwp.

Reply:
I'll clone your branch and test it on macOS and iOS (and maybe Android, that one needs some additional setup from me) later tonight. It would be nice to see the token test show up on each platform since these are definitely cross platform.